packages/aws-client/src/S3.ts

```diff
@@ -958,7 +958,7 @@ export const multipartCopyObject = async (
     );
 
     // Let S3 know that the multi-part upload (copy) is completed
-    await S3MultipartUploads.completeMultipartUpload({
+    return await S3MultipartUploads.completeMultipartUpload({
```

Comment:
I think that we should probably abstract away how we are implementing the `multipartCopyObject` function. If we return the output of `completeMultipartUpload` like we do here, then it's exposing some details that could make it harder to change later.

Rather than returning the result of `completeMultipartUpload`, consider returning `{ etag: response.ETag }`.

Reply:
I did wrestle with that thought as well, but went the route of returning the full result since that's what the rest of the functions here do.  However, since this is not a simple wrapper like the other functions are, I can see how it would make sense here to reduce the exposure.

However, for consistency with what other functions return, which are AWS response objects, should we keep the property name casing consistent by returning `{ ETag }`?